README.md

```diff
@@ -503,7 +507,7 @@ Must be either 'present' or 'absent'.
 
 ##### `name`
 
-The name of the feature.
+The unique name of the feature to manage.
```

Comment:
As above, suggest clarifying that it must be unique as opposed to uniqueness being a intrinsic quality of the property.

ie, the difference between 'the unique name' and 'the name which must be unique' is semantically important.